test/commands/recordcounts.test.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Connection, Org } from '@salesforce/core';
+import { $$, test, expect } from '@salesforce/command/lib/test';
+
+describe('force:limits:recordcounts:display', () => {
+  async function prepareStubs() {
+    $$.SANDBOX.stub(Org.prototype, 'getConnection').callsFake(() => Connection.prototype);
+    $$.SANDBOX.stub(Connection.prototype, 'request').resolves({
+      sObjects: [
+        { count: 34, name: 'Account' },
+        { count: 116, name: 'Contact' },
+      ],
+    });
+  }
+
+  it('queries and aggregates data correctly', () => {
+    test
+      .do(() => prepareStubs())
+      .stdout()
+      .command(['force:limits:recordcounts:display', '--objectnames', 'Account,Contact,Lead', '--json'])
```

Comment:
Also, I'm no programmer, but shouldn't this be "--sobjecttype" (after you make the change to a singular flag name)?

Reply:
Thanks for spotting - yes it should! Fixed.